pkg/api/validation/dynakube/eec.go

```diff
@@ -24,6 +26,31 @@ func extensionControllerImage(_ context.Context, _ *Validator, dk *dynakube.Dyna
 	return ""
 }
 
+func warnIfmultiplyDKwithExtensionsEnabled(ctx context.Context, dv *Validator, dk *dynakube.DynaKube) string {
```

Comment:
```suggestion
func conflictingTenantForExtensions(ctx context.Context, dv *Validator, dk *dynakube.DynaKube) string {
```

Reply:
you could also call it `conflictingApiUrlForExtensions`, to be more precious

Reply:
[(based it on already existing validation func names)](https://github.com/Dynatrace/dynatrace-operator/blob/f85df8f8f7fb174496f39f32bc94cba84c8656cd/pkg/api/validation/dynakube/validation.go#L37-L42)

Reply:
on it!